scripts/post-pip-compile.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -e
+
```

Comment:
@ reviewer: this file was taken from edx-platform. I editted it slightly to take away some unsued paths.